cmd/config_test.go

```diff
@@ -149,16 +183,17 @@ func TestConfigSet(t *testing.T) {
 	var b bytes.Buffer
 	cmdr := DeisCmd{WOut: &b, ConfigFile: cf}
 
-	err = cmdr.ConfigSet("foo", []string{"TRUE=false"})
+	err = cmdr.ConfigSet("foo", []string{"TRUE=false", "SSH_KEY=-----BEGIN OPENSSH PRIVATE KEY-----"})
```

Comment:
Why doesn't it go together in normal usage?

The current docs say that you can set SSH_KEY to either a path or a private key.

The `parseSSHKey` doesn't *really* parse the key, it only checks if it *appears* to be a key by looking at the first line of the key, which is what I'm doing here.

In reality openssh wil happily use a key that just contains the base64 encoded key without any headers/footers, but that way we couldn't differentiate between a single or double base64 encoded key… There's also the golang crypto ssh module's `ParseRawPrivateKey` for actual parsing, but it only supports dss/rsa/ecdsa keys and would exclude ed25519 keys, which are perfectly fine to use here.

Maybe you could be more specific in what you would like me to change about this test.